docs/source/standard/item.rst

```diff
@@ -33,6 +33,8 @@ Schema
     It is required for `classification.scheme` to be `CAV-PS` or `CPV`. The
     `classification.id` should be valid `CAV-PS` or `CPV` code.
 
+    The CPV & CAV-PS must be accurate to at least class (XXXX0000-Y).
```

Comment:
Additionally, there is a validation for the input of these classifiers due to which the accuracy of at least a class (XXXX0000-Y) has to be used.